src/Units/UnitModel.java

```diff
@@ -60,32 +59,29 @@ public static String standardizeID(String input)
     return input.toLowerCase().replaceAll(" ", "_").replaceAll("-", "_");
   }
 
+  public int costBase;
+  public int baseMovePower;
+  public MoveType baseMoveType;
+  public ArrayList<UnitActionFactory> baseActions = new ArrayList<UnitActionFactory>();
+  public int baseCargoCapacity = 0;
+
+  // If you want to modify anything below this line, you're gonna have to implement it into UnitContext
+  public static final int MAXIMUM_HP = 10;
+  public static final int DEFAULT_STAT_RATIO = 100;
   public String name;
   public long role;
-  public Commander CO;
-  public int costBase;
   public double abilityPowerValue;
```

Comment:
Isn't this a function of cost? I didn't realize we just set the unit's AP potential explicitly. One of the things that makes Tech's Steel Hail so OP is that killing the first batch pretty much provides enough power for the next one.

Reply:
It *is* a function of cost. Dual Strike has different charging rules, though, and I wanted to get the data to support it in sooner rather than later.

Reply:
So it's a function of cost, but we've just gone ahead and precalculated it? Might change the BattleMech to be zero in that case.

Reply:
Nope, we haven't precalculated it.
Our current charge (and AW1-2's) is based on unit cost.
AWDS charge uses an independent value defined per unit; we have that value defined, but we don't currently use it.
AWDoR charge (and ours) uses HP damage and ignores cost.